src/Project/MvpSite/rendering/Middleware/CustomGraphQlLayoutServiceHandler.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.Extensions.Configuration;
+using Mvp.Foundation.Configuration.Rendering.AppSettings;
+using Mvp.Foundation.DataFetching.GraphQL;
+using Sitecore.LayoutService.Client.Request;
+using Sitecore.LayoutService.Client.RequestHandlers.GraphQL;
+using System;
+using System.Threading.Tasks;
+
+namespace Mvp.Project.MvpSite.Middleware
+{
+    public class CustomGraphQlLayoutServiceHandler 
+    {
+        private readonly IGraphQLRequestBuilder _graphQLRequestBuilder;
+        private readonly IGraphQLClientFactory _graphQLClientFactory;
+        private readonly MvpSiteSettings _configuration;
+
+        public CustomGraphQlLayoutServiceHandler(IConfiguration configuration, IGraphQLRequestBuilder graphQLRequestBuilder, IGraphQLClientFactory graphQLClientFactory)
+        {
+            _graphQLRequestBuilder = graphQLRequestBuilder;
+            _graphQLClientFactory = graphQLClientFactory;
+            _configuration = configuration.GetSection(MvpSiteSettings.Key).Get<MvpSiteSettings>();
+        }
+
+        public async Task<bool> CheckLayoutExists(
+          SitecoreLayoutRequest layoutrequest)
```

Comment:
Remove new line here, also variable is incorrectly cased, it should he `layoutRequest`